src/components/Modals/ExploreModals/FilterModal/FilterModal.tsx

```diff
@@ -0,0 +1,209 @@
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+import { StatusBar } from 'expo-status-bar';
+import { useState } from 'react';
+import { Platform, SafeAreaView, TouchableOpacity, View } from 'react-native';
+import { useSelector } from 'react-redux';
+import { Button, ButtonType, Icon, Input, Text, ThemedView } from 'src/components';
+import { ScreenTemplate } from 'src/components/templates';
+import { DEFAULT_FILTER_VALUES, DEFAULT_SEARCH_VALUES } from 'src/constants/defaultSearchVelues';
+import { RootStackParamList } from 'src/navigation';
+import { useAppDispatch } from 'src/store';
+import { getFilterSettings, getIsDarkMode } from 'src/store/selectors';
+import { accommodationListActions } from 'src/store/slices';
+import { BLACK, BUTTON_SIZES, WHITE_100, WHITE_200 } from 'src/styles';
+import { IconName, OrderOptions, SearchValues } from 'src/types';
+
+import { styles } from './FilterModal.styles';
+
+const FilterModal = () => {
+  const navigation = useNavigation<NavigationProp<RootStackParamList>>();
+  const colors = useSelector(getIsDarkMode);
+  const dispatch = useAppDispatch();
+  const filter = useSelector(getFilterSettings);
+  const { location, checkInDate, checkOutDate, ...rest } = filter;
+  const [formValues, setFormValues] = useState<SearchValues>(rest);
+
+  const changeOrder = (key: keyof typeof formValues, val: OrderOptions) => {
+    setFormValues((prevFormValues) => ({
+      ...prevFormValues,
+      [key]: val,
+    }));
+  };
+
+  const handleInputChange = (key: keyof typeof formValues, value: number) => {
+    setFormValues((prevFormValues) => ({
+      ...prevFormValues,
+      [key]: value,
+    }));
+  };
+
+  const saveFilterValues = () => {
+    dispatch(accommodationListActions.setFilter(formValues));
+    navigation.goBack();
+  };
+
+  const setFilterToDefault = () => {
+    dispatch(accommodationListActions.setFilter(DEFAULT_SEARCH_VALUES));
+    setFormValues(DEFAULT_FILTER_VALUES);
+  };
+
+  return (
+    <ScreenTemplate
+      style={{
+        flex: 1,
+        justifyContent: 'flex-end',
+        backgroundColor: colors ? 'rgba(255,255,255,0.15)' : 'rgba(0,0,0,0.5)',
+        marginTop: Platform.OS === 'android' ? 30 : undefined,
+      }}
+    >
+      <ThemedView style={styles.container}>
+        <SafeAreaView style={{}}>
```

Comment:
`ScreenTemplate` has a prop named `headerShown`, you can make it as **false** to render the screen with a safe area. So let's remove `SafeAreaView` component here

Reply:
done